docs/javascript/v2/features/pip-mode.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: PIP (Picture-in-Picture) Mode
+nav: 3.37
+---
+
+## Overview
+
+PiP Mode lets the user watch the room video in a small window pinned to a corner of the screen while navigating between apps or browsing content on the main screen.
+
+100ms PiP selects top 4 peers with video enabled and render them in the PiP window.
+
+<video loop="true" autoplay="autoPlay" controls="controls" id="vid" muted>
+    <source src="/docs/guides/pip-demo.mp4" type="video/mp4" />
+</video>
+<br />
+
+## Implemenation
+
+### React projects
+
+For implemenation copy this [folder](https://github.com/100mslive/100ms-web/tree/main/src/components/PIP) to your react project.
```

Comment:
can we please position this as a component and not a bunch of files? it would be useful to show a quick summary of what the different files in the directory mean. we're asking them to take the code and "own it". that means we need to explain what it does

Reply:
we will be exploring webcomponents for this. Till then, it would be like this. will try to pick it in next sprint

Reply:
sounds good